### ISSUE TITLE
Digital Miner has no Ore filter

### ISSUE THREAD

**Issue by Malus-Lupo** [mekanism/Mekanism#5908]
*Please use the search functionality before reporting an issue. Also take a look at the closed issues!*

#### Issue description:
There is no filter for OreDict in the digital miner, There is Item Stack, Tag,  Material, and Mod ID.

#### Steps to reproduce:
1.
2.
3.

#### Version (make sure you are on the latest version before reporting):
**Forge:** 32.1.16
**Mekanism:** 9.9.11.403
**Other relevant version:** 

#### If a (crash)log is relevant for this issue, link it here: (_It's almost always relevant_)

[[gist](https://gist.github.com/) / [pastebin](https://pastebin.com/) / etc link here. Please make sure that it isn't set to expire.]

**Comment by pupnewfster** [comment]
The ore dictionary does not exist anymore, so we have no filter for it as it wouldn't do anything. In the future please check closed issues, as there are multiple ones having to do with the fact of the missing filter. Additionally you could use your best guess, and see if the ore dict filter is gone and there is a new tag filter... that maybe the system got replaced.

**Comment by Malus-Lupo** [comment]
Not sure if you will get this but I am going to give you a little feed back, 1 try to be less rude with your replies? maybe? 2. I looked through the most recent closed issues and didn't see any, I didn't even know there was a search function and it can be missed. 3. If your going to give an answer try to make it a full answer, don't make me out to be stupid for missing a few things I should not have missed, say there is a new system to replace the old system then give 0 info on the new system. I checked the wiki, no info there, I checked closed issues, could not find any there maybe point me in the right direction? Maybe update the wiki? I don't know, maybe just don't make a post where you so clearly wanted to call me stupid short sweet and disrespectful then move on? If that is all you want to do, let some one else answer.
